prime-router/pom.xml

```diff
@@ -30,7 +30,7 @@
         <jackson.version>2.11.3</jackson.version>
         <hapi.version>2.3</hapi.version>
         <jooq.version>3.14.3</jooq.version>
-        <log4j.version>2.8.2</log4j.version>
+        <log4j.version>[2.13.2,)</log4j.version>
         <timestamp>${maven.build.timestamp}</timestamp>
```

Comment:
@RickHawesUSDS , this looks like a typo?

Reply:
Yeah, I can't find any good documentation on the rules for syntax for version numbers in Maven, but this doesn't look correct.

Reply:
Here's some docs https://www.baeldung.com/maven-dependency-latest-version. 

This is not a typo. What this means is >= 2.13.2. It came from the DependaBot folks.

Reply:
It still _looks like_ a typo!   ;)